hack/test-integration.sh

```diff
@@ -33,7 +33,16 @@ if [ "$COVER" -eq 1 ]; then
 fi
 
 # Run the tests and collate the results.
-bats -t $ROOT/test/*.bats
+TESTS=""
+for item in ${BATS[@]};
```

Comment:
Maybe we should make this passed by arguments to the script? I only say this because "typed" environment variables have always scared the crap out of me.